gitlab/misc.sh

```diff
@@ -64,6 +64,23 @@ function validate_schemas_opa() {
   done
 }
 
+function validate_schemas_pluto() {
+  local dir
+  dir=${1-"environments"}
+
+  PLUTO_K8S_VERSION="${PLUTO_K8S_VERSION:-1.16.0}"
+
+  for cluster in $(find $dir -type f -name kustomization.yaml -exec dirname {} \;)
+  do
+    echo "# ---------------------- #"
+    echo "# Validating Manifests with Pluto for Cluster $cluster #"
+    echo "# ---------------------- #"
+    render=$(mktemp tmp.XXXXXXXXXX.yaml)
+    kustomize build $cluster > $render
```

Comment:
I haven't checked, but could it be that we are rendering multiple times here for different tests?

Ideally we would only render manifests once, then run all checks we need across them.

Reply:
Yeah that does happen, but don't different GitLab CI steps run in different containers? In which case we'd need to re-render for each validation anyway.

Reply:
yeah that's true 

don't forget though that we already have the rendered manifests inthe rendered directory ... and we check before this step that they are the same as the live rendering in the golden diff so we could skip this renderending.

That would also make the installation of kustomize / kubecfg irrelevant for this job

Reply:
Have changed the kubecfg/opa/pluto validation checks to use the rendered manifests from the repo. Should speed things up a little bit!